### PR TITLE
Shaun lig 1840 fix lightly unmocked tests

### PR DESCRIPTION
**Pull request by shaundaley39**
Unmocked test changes to get them passing again for release.

Passed:
https://github.com/lightly-ai/lightly/actions/runs/3151260006

### CODE REVIEW

File: tests/UNMOCKED_end2end_tests/run_all_unmocked_tests.sh [hunk 1]
```diff
@@ -21,43 +23,46 @@ python tests/UNMOCKED_end2end_tests/create_custom_metadata_from_input_dir.py $IN
...
+# lightly-upload token=$TOKEN input_dir=$INPUT_DIR new_dataset_name=test_unmocked_cli_3_${DATE_TIME}
 ((NUMBER_OF_DATASETS=NUMBER_OF_DATASETS+1))
 
 echo "############################### Test 4"
-lightly-magic token=$TOKEN input_dir=$INPUT_DIR trainer.max_epochs=0 new_dataset_name=test_unmocked_cli_4
-lightly-magic token=$TOKEN input_dir=$INPUT_DIR trainer.max_epochs=0 new_dataset_name=test_unmocked_cli_4
+lightly-magic token=$TOKEN input_dir=$INPUT_DIR trainer.max_epochs=0 new_dataset_name=test_unmocked_cli_4_${DATE_TIME}
+# This would fail, since a dataset with this name already exists
+# lightly-magic token=$TOKEN input_dir=$INPUT_DIR trainer.max_epochs=0 new_dataset_name=test_unmocked_cli_4_${DATE_TIME}
 ((NUMBER_OF_DATASETS=NUMBER_OF_DATASETS+1))
```
Comment (guarin) [inline tests/UNMOCKED_end2end_tests/run_all_unmocked_tests.sh:43]: I think this test can be removed as it is now identical to Test 1.

File: tests/UNMOCKED_end2end_tests/run_all_unmocked_tests.sh [hunk 2]
```diff
@@ -21,43 +23,46 @@ python tests/UNMOCKED_end2end_tests/create_custom_metadata_from_input_dir.py $IN
...
+lightly-upload token=$TOKEN input_dir=$INPUT_DIR new_dataset_name=test_unmocked_cli_7_${DATE_TIME} upload=metadata custom_metadata=$CUSTOM_METADATA_FILENAME
 ((NUMBER_OF_DATASETS=NUMBER_OF_DATASETS+1))
 
 echo "############################### Test 8"
-lightly-upload token=$TOKEN input_dir=$INPUT_DIR new_dataset_name=test_unmocked_cli_8 upload=metadata
-lightly-upload token=$TOKEN new_dataset_name=test_unmocked_cli_8 custom_metadata=$CUSTOM_METADATA_FILENAME
+lightly-upload token=$TOKEN input_dir=$INPUT_DIR new_dataset_name=test_unmocked_cli_8_${DATE_TIME} upload=metadata custom_metadata=$CUSTOM_METADATA_FILENAME
+# This would fail, since a dataset with this name already exists
+# lightly-upload token=$TOKEN new_dataset_name=test_unmocked_cli_8_${DATE_TIME} custom_metadata=$CUSTOM_METADATA_FILENAME
 ((NUMBER_OF_DATASETS=NUMBER_OF_DATASETS+1))
```
Comment (guarin) [inline tests/UNMOCKED_end2end_tests/run_all_unmocked_tests.sh:61]: Identical to Test 7.

**Review by guarin — APPROVED** [review]
I believe some of the tests are now redundant. But feel free to merge anyways.

File: tests/UNMOCKED_end2end_tests/run_all_unmocked_tests.sh (hunk 1 above)
Comment (shaundaley39) [reply tests/UNMOCKED_end2end_tests/run_all_unmocked_tests.sh]: dropped

**Pull request merged**